ui/src/errors/components/ErrorPage.jsx

```diff
@@ -19,6 +19,13 @@ class ErrorPage extends Component {
             </Col>
           </Row>
         )}
+        {url && (
+          <Row>
+            <Col span={24}>
+              <p className="f5 normal">Redirected from: {url}</p>
```

Comment:
shouldn't be a link back? instead of string?

Reply:
I talked with Micha about how it should work and purposely put the url string here instead of our standard go back link, as it makes it easy to copy the broken page url.

Reply:
why not both? a link which has as a text the url?